main/settings.py

```diff
@@ -309,45 +310,6 @@
     "profile",  # this avoids an error because profile is a related model
 ]
 
-SOCIAL_AUTH_PIPELINE = (
-    # Checks if an admin user attempts to login/register while hijacking another user.
-    "authentication.pipeline.user.forbid_hijack",
-    # Get the information we can about the user and return it in a simple
-    # format to create the user instance later. On some cases the details are
-    # already part of the auth response from the provider, but sometimes this
-    # could hit a provider API.
-    "social_core.pipeline.social_auth.social_details",
-    # Get the social uid from whichever service we're authing thru. The uid is
-    # the unique identifier of the given user in the provider.
-    "social_core.pipeline.social_auth.social_uid",
-    # Verifies that the current auth process is valid within the current
-    # project, this is where emails and domains whitelists are applied (if
-    # defined).
-    "social_core.pipeline.social_auth.auth_allowed",
-    # Checks if the current social-account is already associated in the site.
-    "social_core.pipeline.social_auth.social_user",
-    # Associates current social details with another user account with same email.
-    "social_core.pipeline.social_auth.associate_by_email",
-    # Send a validation email to the user to verify its email address.
-    # Disabled by default.
-    "social_core.pipeline.mail.mail_validation",
-    # # Generate a username for the user
-    # # NOTE: needs to be right before create_user so nothing overrides the username
-    # "authentication.pipeline.user.get_username",
-    # Create a user account if we haven't found one yet.
-    "social_core.pipeline.user.create_user",
-    # Create the record that associates the social account with the user.
-    "social_core.pipeline.social_auth.associate_user",
-    # Populate the extra_data field in the social record with the values
-    # specified by settings (and the default ones like access_token, etc).
-    "social_core.pipeline.social_auth.load_extra_data",
-    # Update the user record with any changed info from the auth service.
-    "social_core.pipeline.user.user_details",
-    # Create a favorites list for new users
-    "authentication.pipeline.user.user_created_actions",
-    # redirect new users to onboarding
-    "authentication.pipeline.user.user_onboarding",
-)
```

Comment:
This didn't seem to be getting used even without keycloak and apisix

Reply:
PS the final pipeline step is supposed to redirect the user to "/onboarding" but that isn't currently happening on RC with new users I created there, even though `MITOL_NEW_USER_LOGIN_URL=https://rc.learn.mit.edu/onboarding` - should it be?  Haven't tried it on production yet.